Changelog.md

```diff
@@ -71,6 +71,7 @@ Compiler Features:
 Bugfixes:
  * Tests: Fix chain parameters to make ipc tests work with newer versions of cpp-ethereum.
  * Code Generator: Fix allocation of byte arrays (zeroed out too much memory).
+ * Code Generator: Nested arrays returned by library functions were always zeroed up to version 0.4.21. The bug was fixed in version 0.4.22 by using ``pragma experimental ABIEncoderV2``. If the ``ABIEncoderV2`` is not used, the compiler issues a TypeError on the nested array returned by the library function.
```

Comment:
Not sure we need to have this in the bugfixes section since we didn't do anything in regards to it in 0.5.0.

Maybe worth adding a comment somewhere on the top that a new entry was added to the bug list?

Reply:
Yea it doesn't really fit in any of the lists